src/main/java/net/sf/mzmine/taskcontrol/impl/TaskControllerImpl.java

```diff
@@ -92,6 +93,14 @@ public void addTasks(Task tasks[]) {
     }
 
     public void addTasks(Task tasks[], TaskPriority priority) {
+	// Track module usage
+	String taskClass = tasks[0].getClass().getName();
```

Comment:
tasks[0] is going to throw an exception if the array is empty (see the code comment a few lines below). 
Generally in Java, using anything[0] signifies there is a programming problem.
In this case, you should probably go through the array and make a Set of all Classes of the tasks - because there is no guarantee that the tasks[] array contains tasks of the same class.